ujit_asm.c

```diff
@@ -1629,3 +1629,9 @@ void xor(codeblock_t* cb, x86opnd_t opnd0, x86opnd_t opnd1)
         opnd1
     );
 }
+
+// LOCK - lock prefix for atomic shared memory operations
+void gen_lock_prefix(codeblock_t* cb)
```

Comment:
Can you rename that to `cb_write_lock_prefix` to match the other functions in the file?